### ISSUE TITLE
Some sort of VPN 

### ISSUE THREAD

**Issue by MichalAsik** [Botspot/pi-apps#586]
Can this app run on RPi? If so, please provide a link to a tutorial if applicable.  
Is this app something many GUI users would find useful?  
Is this app installable with `sudo apt install`?  

If you've created a zip file for this app, upload it here.

**Comment by chunky-milk** [comment]
@MichalAsik which vpn client are you asking about? I know that the nordvpn Linux client works well on the Pi.

**Comment by mobilegmYT** [comment]
@MichalAsik different vpn's have their own methods for rpi support, you can google install instructions for your vpn, and half of them use deb packages.

**Comment by GabeCoder** [comment]
Wow, this is a really useful application if you want to privately chat with others on Zoom or Google Meet. This is also really useful since one of the Pi Apps users actually lives in China. (Not Taiwan, the communist and censored China)
I don't know if this is legal, but I will support this.

**Comment by mobilegmYT** [comment]
> Wow, this is a really useful application if you want to privately chat with others on Zoom or Google Meet. This is also really useful since one of the Pi Apps users actually lives in China. (Not Taiwan, the communist and censored China)
> I don't know if this is legal, but I will support this.

I mean I don't see much purpose with this considering there are tons of vpns out there, and it isn't particularly hard to follow instructions for your VPN.

**Comment by Botspot** [comment]
> since one of the Pi Apps users actually lives in China.

Oh I sure hope Pi-Apps has more than one Chinese user!

**Comment by Botspot** [comment]
I'm open to the idea of creating a new subcategory within "Internet" for all common VPNs that can be installed on RPi. This would take a lot of work and research to accomplish though, and I don't have the knowledge or the willpower to do it.

But if someone else wants to, I'll do my best to be supportive.
Closing this issue now. If someone sees this and would like to add a VPN or two, it should take place in new individual issue(s).
